airflow/api_connexion/endpoints/health_endpoint.py

```diff
@@ -14,13 +14,35 @@
 # KIND, either express or implied.  See the License for the
 # specific language governing permissions and limitations
 # under the License.
-
-# TODO(mik-laj): We have to implement it.
-#     Do you want to help? Please look at: https://github.com/apache/airflow/issues/8144
+from airflow.api_connexion.schemas.health_schema import health_schema
+from airflow.jobs.scheduler_job import SchedulerJob
 
 
 def get_health():
     """
-    Checks if the API works
+    Return the health of the airflow scheduler and metadatabase
     """
-    return "OK"
+    HEALTHY = "healthy"  # pylint: disable=invalid-name
+    UNHEALTHY = "unhealthy"  # pylint: disable=invalid-name
```

Comment:
If you will move those out of the function then there will be no need to disable pylint :)